### ISSUE TITLE
Simplify Configs

### ISSUE THREAD

**Issue by Licini** [compas-dev/compas_viewer#108]
Currently there exist a series of Config related classes. For instance for`Camera` there is a `CameraConfig`, then even a `CameraConfigType`. For me these layers feels very redundant. Why not inherent all these objects from `Data` class, then data needed for setting up the camera can be handled by `__data__` and `__from_data__`,  for type checking we use `DATASCHEMA`. @tomvanmele What do you think?

**Comment by tomvanmele** [comment]
yes, i like the idea of configurators, but the system is indeed to complicated and not very transparent (e.g. not very clear why certain things are configured in certain places). also the fact that defaults are loaded from "data" files doesn't make sense to me. the user should be able to define defaults manually...

**Comment by Licini** [comment]
Agree.... maybe makes more sense to meet somewhere in middle. I like how it is done in notebook, still having "configurators" but just one layer using DataClass, not two. Will make a PR on this direction
